include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -754,6 +766,12 @@ Operator<dim, Number>::compute_initial_acceleration(VectorType &       accelerat
 
   // invert mass operator to get acceleration
   mass_solver->solve(acceleration, rhs);
+
+  // revert scaling factor to initialized value
+  if(param.large_deformation)
+    elasticity_operator_nonlinear.set_scaling_factor_mass_operator(scaling_factor_mass);
```

Comment:
I would write each of these lines directly after evaluating the elasticity operator in the respective if/else-branches above so that it is more clear for which parts of the code this is relevant.